src/sql/arrow_sql_gen/postgres.rs

```diff
@@ -721,6 +775,15 @@ fn map_column_type_to_data_type(column_type: &Type) -> Option<DataType> {
             DataType::Boolean,
             true,
         )))),
+        Type::BYTEA_ARRAY => Some(DataType::List(Arc::new(Field::new(
```

Comment:
Would you be able to add a test for this so we can ensure it doesn't break in the future? Adding a new test method to https://github.com/datafusion-contrib/datafusion-table-providers/blob/main/tests/postgres/mod.rs and following the existing pattern for how `test_arrow_postgres_roundtrip` works should be fairly straightforward.

Reply:
I have added tests.